README.md

```diff
@@ -75,9 +75,9 @@ If you have live server extension
 
 👥 Authors 
 
-- GitHub:https://github.com/Gardimy
-- Twitter:https://twitter.com/
-- LinkedIn:https://www.linkedin.com/in/gardimy-charles-40b795212/
+- GitHub: [@githubhandle](https://github.com/Hamdullah-Andar)
+- Twitter: [@twitterhandle](https://twitter.com/HamdullahAndar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hamdullah-andar-98b35894/)
```

Comment:
- [ ] This is a pair programming project, so your partner should make a contribution to the code evidenced by the _**git commit**_ from them. Please ensure your partner `gardimy-charles` make contirbution. 
- [ ] Therefore, you should include their social media handles in the ReadMe